docs/disable_with_comments.rst

```diff
@@ -89,7 +94,7 @@ For instance:
  key: value 2
 
  - This line is waaaaaaaaaay too long but yamllint will not report anything about it.
-   This line will be checked by yamllint.
+   This line won't be checked by yamllint.
```

Comment:
I propose to remove the whole line `This line won't be checked by yamllint.` entirely (it doesn't help for clarity).